concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -681,8 +681,8 @@ Hale-1973-1798-679	House_and_Home-21	post	n	06a.21	281	POST
 Hale-1973-1798-680	House_and_Home-22	bamboo ties (strips for building)	n	06a.22		
 Hale-1973-1798-681	House_and_Home-23	scaffold	n	06a.23		
 Hale-1973-1798-682	House_and_Home-24	ladder	n	06a.24	1464	LADDER
-Hale-1973-1798-683	House_and_Home-25	veranda	n	06a.25		
-Hale-1973-1798-684	House_and_Home-26	doorway	n	06a.26		
+Hale-1973-1798-683	House_and_Home-25	veranda	n	06a.25	3269	PORCH
+Hale-1973-1798-684	House_and_Home-26	doorway	n	06a.26	1567	DOOR
```

Comment:
Surprisingly, this is correct if we consider the [defintion of `1567 DOOR` in concepticon](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L1568): "An opening, or passage in a fence or wall; the entrance through which you enter or leave a room or building." vs. the [defintion of "doorway" in an online English dictionary](https://www.merriam-webster.com/dictionary/doorway): "the opening that a door closes"

Reply:
This should be discussed in a separate issue. We have some mappings of _doorway_ to DOOR but other cases are unmapped. @mathildavz Could you open an issue where we discuss the details?

Reply:
So should I leave it mapped for now? @AnnikaTjuka

Reply:
I'd unmap this for now.